docs/dev/reference/dca/callbacks.md

```diff
@@ -774,7 +774,7 @@ use Contao\DataContainer;
 #[AsCallback('tl_content', 'fields.text.attributes')]
 class AttributesCallback
 {
-    public function __invoke(array $attributes, DataContainer|null $dc = null): array
+    public function __invoke(array $attributes, ?DataContainer $dc = null): array
```

Comment:
This needs to be reverted, the parameter is already nullable (in the Contao code style).